rs/replicated_state/Cargo.toml

```diff
@@ -36,6 +36,7 @@ maplit = "1.0.2"
 nix = { workspace = true }
 rand = { workspace = true }
 rand_chacha = { workspace = true }
+rayon = "1.5.1"
```

Comment:
Can we use `workspace = true` here? As far as I know this is the preferred way of specifying versions.

Reply:
Thanks, fixed now.